Microsoft.Toolkit.Uwp.UI.Controls.Core/Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj

```diff
@@ -12,12 +12,34 @@
         - ImageEx: Images are downloaded asynchronously showing a load indicator and can be stored in a local cache.
         - InAppNotification: Show local notifications in your application.
         - Loading: Helps to show content with animation to the user while the app is doing some calculation.
+        - Menu: TODO
```

Comment:
Needs description.

Reply:
Menu is deprecated, so let's not call it out.

Reply:
Removed the line